tests/spread/general/store/task.yaml

```diff
@@ -18,6 +18,9 @@ prepare: |
     exit 1
   fi
 
+  # components are only supported on the edge channel
+  snap refresh snapd --edge
```

Comment:
we do not need this with my fix

Reply:
10-4, I'll drop this when your PR lands.